src/bionics.cpp

```diff
@@ -1561,13 +1561,12 @@ float Character::get_effective_efficiency( int b, float fuel_efficiency )
  * @param p the player
  * @param bio the bionic that is meant to be recharged.
  * @param amount the amount of power that is to be spent recharging the bionic.
- * @param factor multiplies the power cost per turn.
  * @return indicates whether we successfully charged the bionic.
  */
-static bool attempt_recharge( Character &p, bionic &bio, units::energy &amount, int factor = 1 )
+static bool attempt_recharge( Character &p, bionic &bio, units::energy &amount )
 {
     const bionic_data &info = bio.info();
-    units::energy power_cost = info.power_over_time * factor;
```

Comment:
What was this thing doing and why are you removing it? It might very well be a good thing but this seem unrelated to the other changes

Reply:
There is only one place where that function is called and the variable is always set to 1 (that is also removed).

It is unrelated. I just noticed it while looking at how normal power cost works and decided to throw it away while working here.

Probably old code for power cost modifiers based on environmental conditions, status effects, faulty bionics, etc. (at least that is what the comment says. The comment should probably be removed too).